src/dxvk/dxvk_extensions.h

```diff
@@ -292,8 +292,9 @@ namespace dxvk {
    * used by DXVK if supported by the implementation.
    */
   struct DxvkInstanceExtensions {
+    DxvkExt extDebugUtils                   = { VK_EXT_DEBUG_UTILS_EXTENSION_NAME,                      DxvkExtMode::Optional };
```

Comment:
We shouldn't enable this by default. It's not free.

Reply:
Maybe use an env var for this like we do in VKD3D.

Reply:
Or a dxvk.conf option.

Reply:
I'd prefer to go with an environment variable for toggling this on/off, since it's a bit more lightweight to enable while changing between different applications. How does a boolean `DXVK_PERF_EVENTS` sound?

Reply:
Sounds good.